docs/modules/sections/pages/part-numbers-and-labels.adoc

```diff
@@ -28,15 +28,15 @@ Part I. Defensive Operations
 Part II. Managing Werewolves
 ....
 
-[#part-label]
-== Customize book part labels
+[#part-signifier]
+== Customize book part signifiers
 
 The label _Part_ is automatically added to the beginning of the part titles when the `partnums` attribute is set.
-You can modify the label by defining the `part-label` attribute in the header of your book.
+You can modify the label by defining the `part-signifier` attribute in the header of your book.
 
 [source]
 ----
-:part-label: Component
+:part-signifier: Component 
```

Comment:
Can you remove the trailing space on this line?

Reply:
Done. Thanks